pom.xml

```diff
@@ -32,18 +32,31 @@
 		</repository>
 	</repositories>
 
-	<modules>
-		<module>utils</module>
-		<module>conformance</module>
-		<module>standardization</module>
-		<module>dataModel</module>
-		<module>dao</module>
-		<module>menas</module>
-		<module>migrations</module>
-		<module>external/kafka</module>
-		<module>external/fixedWidth</module>
-		<module>selenium</module>
-	</modules>
+	<profiles>
```

Comment:
Let's leave the modules as they were without `selenium` and have a profile which specifies the selenium module.
The profiles usually go at the end of the pom.xml.